src/modules/launcher/PowerLauncher/MainWindow.xaml

```diff
@@ -32,14 +32,14 @@
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="Auto"/>
         </Grid.RowDefinitions>
-        <Border Margin="10" Grid.Row="0" BorderBrush="Black" BorderThickness="5" CornerRadius="4">
+        <Border x:Name="SearchBoxBorder" Margin="10" Grid.Row="0" BorderThickness="3" CornerRadius="4">

```

Comment:
Please change the BorderThickness to atleast 4, it looks wrong at the corners if it is smaller than the CornerRadius

Reply:
Sure!! Thanks for you suggestions!!